bcda/auth/models.go

```diff
@@ -63,3 +62,22 @@ func GetACOByClientID(clientID string) (models.ACO, error) {
 	}
 	return aco, err
 }
+
+// RevokeSystemKeyPair soft deletes the specified encryption key so that it can no longer be used
+func RevokeSystemKeyPair(encryptionKeyID uint) error {
```

Comment:
The CLI would need an intermediate step to look up the key ID before calling this, right? Asking because the function for generating the key was meant to take the client ID, based on the mini tech spec. I wonder if it should be the same here.

Reply:
changed to systemID